R/utils.R

```diff
@@ -181,10 +181,14 @@ ask <- function(...) {
 # Deal with proxy-related connection issues
```

Comment:
Just a little more detail. Replace "Deal..." with something like "Catch network timeout error generated when dealing with proxy-related connection issues and fail with a information error message of where to download HYDAT"